spec/lib/karafka/pro/active_job/current_attributes_spec.rb

```diff
@@ -0,0 +1,112 @@
+# frozen_string_literal: true
```

Comment:
I duplicated the spec for Pro because the Pro Dispatcher overrides the dispatch method and handles serialization itself. I wanted to make sure both Dispathers and Consumers (Pro and non-pro) used CurrentAttribues when enabled.